docs/content/2.deploy/providers/firebase.md

```diff
@@ -90,3 +92,36 @@ Deploying to Firebase Hosting is a simple matter of just running the `firebase d
 NITRO_PRESET=firebase yarn build
 firebase deploy
 ```
+
+## Firebase Functions V2
+
+**Preset:** `firebaseV2` ([switch to this preset](/deploy/#changing-the-deployment-preset))
+
+This preset will deploy to [firebase functions V2](https://firebase.google.com/docs/functions/beta/get-started) - the 2nd generation of cloud functions.
```

Comment:
This link doesn't work anymore, maybe we could link to https://firebase.google.com/docs/functions/version-comparison